src/scene.rs

```diff
@@ -219,6 +223,17 @@ impl Scene {
     }
 }
 
+impl From<Encoding> for Scene {
+    fn from(encoding: Encoding) -> Self {
+        // TODO: Properly update the estimator for the encoding.
```

Comment:
And this is why I'd make it a `from_encoding` method, to document that the bump estimator will undercount items added in this method

Reply:
In other conversations, it has already been said that the implementation strategy of the estimator will be changing and this issue would go away.

Reply:
Note that if you're adding a TODO to the code, it's usually good practice to join a link to a ticket (usually a github issue) tracking down whatever needs to be done to remove that TODO.

Reply:
It matches the existing TODO comments elsewhere in the file for the same feature. (Also, out of roughly 80 TODOs in this repo, approximately 2 have an issue number in them.)

Reply:
We can do a comments pass at some point, but I think at least new comments should follow that practice.

Reply:
We could point to https://github.com/linebender/vello/issues/530#issuecomment-2016585267

Reply:
Or indeed https://github.com/linebender/vello/issues/541